test/e2e/framework/node/wait.go

```diff
@@ -160,32 +156,42 @@ func WaitForNodeSchedulable(ctx context.Context, c clientset.Interface, name str
 	return false
 }
 
+func WaitForNodeHeartbeatAfter(ctx context.Context, c clientset.Interface, name string, after metav1.Time, timeout time.Duration) bool {
```

Comment:
There's no need for returning the boolean.

The failure handler will abort with a panic, so there's never a situation where this function returns false.